data/config/datasets/adaguc.tests.solarterminator.xml

```diff
@@ -22,6 +22,7 @@
     <DataPostProc algorithm="solarterminator"/>
     <Variable>solarterminator</Variable>
     <Styles>soltstyle</Styles>
+    <Dimension interval="PT1M">time</Dimension>
```

Comment:
I would put 10M here (1M looks like overkill for something like the solar terminator)

Reply:
Updated!